src/templates/app/resources/economic_calendar.jsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+
+const EconomicCalendar = () => (
+    <div id='economic_calendar' className='static_full'>
+        <div className='economic-calendar-header gr-row'>
+            <div className='gr-12 gr-centered'>
```

Comment:
why add `gr-12` here? it negates the `gr-row` on top. we can remove both `gr-row` and `gr-12`, and even `gr-centered` as I don't see the point